packages/core/src/openapi/create-open-api-document.spec.ts

```diff
@@ -29,6 +29,32 @@ describe('createOpenApiDocument', () => {
     strictEqual(document.openapi, '3.0.0');
   });
 
+  describe('path validation', () => {
+    it('consistient canonicalisation', () => {
+      strictEqual(canonicalisePath('/test/path/{parameter}'), '/test/path/#');
+      strictEqual(canonicalisePath('/test/path/{parameter}/sub/{sub_parameter}'), '/test/path/#/sub/#');
+    });
+
+    it('duplicate paths throws error', () => {
+      try {
+        throwErrorIfDuplicatePaths({
+          '/test/path/{parameter_1}': {},
+          '/test/path/{parameter_2}': {}
+        });
+        throw new Error('Expected an exception here');
+      } catch (err) {
+        notStrictEqual(err.message, 'Expected an exception here');
```

Comment:
I added this line here. Otherwise the test was always successful because all errors (thrown in line 40 or line 44) were caught.

Reply:
ah true, thanks